### PR TITLE
Simplify deploy workflow variables

### PR DESCRIPTION
**Pull request by NullableInt**
I am unsure if I should simplify the `docker run` to use `my_awesome_image` to make it easier to read, but I think I want to keep it to ensure things are working correctly.

### CODE REVIEW

File: .github/workflows/docker_build_test_publish.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
         username: swedbankpay
```
Comment (asbjornu) [inline .github/workflows/docker_build_test_publish.yml:33]:
```suggestion
        username: ${{ github.repository_owner }}
```

File: .github/workflows/docker_build_test_publish.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: "${{ secrets.DOCKERHUB_PASSWORD }}"  # you don't need to manually set this secret. GitHub does it on your behalf
```
Comment (asbjornu) [inline .github/workflows/docker_build_test_publish.yml:49]:
```suggestion
        password: "${{ secrets.DOCKERHUB_PASSWORD }}"
```

File: .github/workflows/docker_build_test_publish.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ jobs:
         username: swedbankpay
         password: "${{ secrets.GITHUB_TOKEN }}"  # you don't need to manually set this secret. GitHub does it on your behalf
```
Comment (asbjornu) [inline .github/workflows/docker_build_test_publish.yml:34]:
```suggestion
        password: "${{ secrets.GITHUB_TOKEN }}"
```

File: .github/workflows/docker_build_test_publish.yml [hunk 4]
```diff
@@ -47,7 +47,7 @@ jobs:
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: "${{ secrets.DOCKERHUB_PASSWORD }}"  # you don't need to manually set this secret. GitHub does it on your behalf
-        image_name: swedbankpay/jekyll-plantuml
+        image_name: jekyll-plantuml
         image_tag: ${{ steps.version.outputs.version }}
         dockerfile: ".docker/Dockerfile"
         push_image_and_stages: docker run --tty --volume ${{ github.workspace }}/test/jekyll-plantuml:/srv/jekyll swedbankpay/jekyll-plantuml:${{ steps.version.outputs.version }} jekyll build
```
Comment (asbjornu) [inline .github/workflows/docker_build_test_publish.yml:53]: The hostname is optional here because we're pushing to `docker.com`?

Comment (NullableInt) [reply .github/workflows/docker_build_test_publish.yml:53]: Yup, that's the default. I can add it to be explicit if wanted.

Comment (asbjornu) [reply .github/workflows/docker_build_test_publish.yml:53]: Nah, I think it's okay. As long as it works. 😄

Comment (NullableInt) [reply .github/workflows/docker_build_test_publish.yml:53]: The last deploy worked at least 🙏

**Pull request merged**